### PR TITLE
CVA-244/variant interpretation logs

### PR DESCRIPTION
**Pull request by andyb3**
This change is [Reviewable](https://reviewable.io/reviews/genomicsengland/pycipapi/39)

### CODE REVIEW

File: pycipapi/cipapi_client.py [hunk 1]
```diff
@@ -126,6 +127,14 @@ def submit_clinical_report_raw(self, payload, partner_id, analysis_type, report_
         url = self.build_url(self.url_base, self.CR_ENDPOINT, partner_id, analysis_type, report_id) + '/'
         return self.post(url, payload, params=params)
 
+    def submit_variant_interpretation_logs_raw(self, payload, case_id, case_version, **params):
+        case_id_version = "{ir_id}-{ir_version}".format(
+            ir_id=case_id,
+            ir_version=case_version
+        )
+        url = self.build_url(self.url_base, self.IR_ENDPOINT, case_id_version, 'variant-interpretation-log') + '/'
```
Comment (cboustred) [inline pycipapi/cipapi_client.py:135]: is IR_ENDPOINT correct?

File: pycipapi/cipapi_client.py [hunk 2]
```diff
@@ -126,6 +127,14 @@ def submit_clinical_report_raw(self, payload, partner_id, analysis_type, report_
         url = self.build_url(self.url_base, self.CR_ENDPOINT, partner_id, analysis_type, report_id) + '/'
         return self.post(url, payload, params=params)
 
+    def submit_variant_interpretation_logs_raw(self, payload, case_id, case_version, **params):
```
Comment (cboustred) [inline pycipapi/cipapi_client.py:130]: _logs_ or _log_ ?

e.g. if its a single log for a single case then maybe "log" is better than "logs" ?

File: pycipapi/cipapi_client.py (hunk 1 above)
Comment (andyb3) [reply pycipapi/cipapi_client.py:135]: Yeah, the vil endpoint sits underneath the IR endpoint 
image

File: pycipapi/cipapi_client.py (hunk 2 above)
Comment (andyb3) [reply pycipapi/cipapi_client.py:130]: It's for a single case, but you can post a list of logs (one for each variant)

**Review by cboustred — APPROVED** [review]
LGTM

**Pull request merged**